src/library_async.js

```diff
@@ -533,6 +533,17 @@ mergeInto(LibraryManager.library, {
     });
   },
 
+  emscripten_load_secondary_module__sig: 'v',
+  emscripten_load_secondary_module: async function() {
+    Module['asm']['load_secondary_module_status'].value = 1;
```

Comment:
Maybe add a comment here, as it looks like this is used from code in another repo (binaryen),
```suggestion
    // Mark the module as loading for the wasm module (so it doesn't try to load it again).
    Module['asm']['load_secondary_module_status'].value = 1;
```